src/main/java/com/javadiscord/javabot/service/AutoMod.java

```diff
@@ -85,6 +108,15 @@ private void checkContentAutomod(@Nonnull Message message) {
         if (matcher.find()) {
             warn(message, message.getMember(), "Automod: Advertising");
         }
+        final String messageRaw = message.getContentRaw();
+        if (messageRaw.startsWith("http://") || messageRaw.startsWith("https://")) {
+            // only do it for a links, so it won't iterate for each message
+            for (String spamUrl : spamUrls) {
+                if (messageRaw.contains(spamUrl)){
+                    new Ban().ban(message.getMember(), "Scam");
```

Comment:
You're still banning people although we said we would warn them

And please use the same reason-naming-scheme we used everywhere else for AutoMod, so it should be `Automod: Suspicious Link` or something similar

Reply:
People are stupid and they will post links on accident or because they want to know whether it's harmful or not and just don't know the damages it might cause to others - we don't ban people if they post a message with that link asking whether it's safe or not - we tell them not to do it again and in some cases warn them.

Reply:
And something else regarding this part; You should also delete the message that was sent

Reply:
Alright Done